test/test_helper.rb

```diff
@@ -49,3 +49,10 @@ class ActionDispatch::IntegrationTest
 class SystemTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 end
+
+Shoulda::Matchers.configure do |config|
```

Comment:
No changelog :( Why are we adding this?
Can you please also undo change of BUNDLED WITH?

Reply:
Updated commit to explain the config block here and undo the bundler version bump.